style/layer/highway_shield.js

```diff
@@ -11,9 +11,19 @@ function routeConcurrency(num) {
   ];
 }
 
+function routeSpacer(num) {
+  return [
+    "case",
+    ["!=", ["get", "route_" + num], null],
+    ["literal", "  "],
+    ["literal", ""],
```

Comment:
This version of the text route spacer is glitching too.

![Screenshot 2022-01-07 12 07 51](https://user-images.githubusercontent.com/23022/148580295-c5bfcc86-9ff7-4727-ae07-c85363a4fb9a.png)
.